bundles/org.smarthomej.binding.viessmann/src/main/java/org/smarthomej/binding/viessmann/internal/handler/ViessmannBridgeHandler.java

```diff
@@ -162,7 +179,7 @@ public void initialize() {
             setConfigInstallationGatewayId();
         }
 
-        if (errorChannelsLinked()) {
+        if (errorChannelsLinked() && !config.disablePolling) {
```

Comment:
```suggestion
        if (!config.disablePolling && errorChannelsLinked()) {
```

Might not be the biggest improvement, but the `!config.disablePolling` check is cheap compared to the `isLinked` check, so it's better to do it first.